test/integration/analytics_int_test.go

```diff
@@ -177,6 +207,58 @@ func (suite *AnalyticsIntegrationTestSuite) TestActivateEvents() {
 	suite.assertSequentialEvents(events)
 }
 
+func (suite *AnalyticsIntegrationTestSuite) TestExecEvents() {
+	suite.OnlyRunForTags(tagsuite.Analytics, tagsuite.Critical)
+
+	/* TEST SETUP */
+
+	ts := e2e.New(suite.T(), true)
+	defer ts.Close()
+
+	namespace := "ActiveState-CLI/Alternate-Python"
+	commitID := "efcc851f-1451-4d0a-9dcb-074ac3f35f0a"
+
+	// We want to do a clean test without an activate event, so we have to manually seed the yaml
+	url := fmt.Sprintf("https://platform.activestate.com/%s?branch=main&commitID=%s", namespace, commitID)
+	suite.Require().NoError(fileutils.WriteFile(filepath.Join(ts.Dirs.Work, "activestate.yaml"), []byte("project: "+url)))
```

Comment:
You'll probably want to change this to use `ts.PrepareProject()` when merging with v42: https://github.com/ActiveState/cli/blob/version/0-42-0-RC1/internal/testhelpers/e2e/session.go#L304.